cypress.config.js

```diff
@@ -1,21 +1,22 @@
-const { defineConfig } = require("cypress");
-const faker = require('faker');
-
-module.exports = defineConfig({
-  e2e: {
-    baseUrl: 'https://conduit.mate.academy/',
-    setupNodeEvents(on, config) {
-      on('task', {
-        generateUser() {
-          const email = faker.internet.email();
-          const randomNumber = Math.floor(Math.random(1000) * 1000);
-          return {
-            username: faker.name.firstName() + randomNumber,
-            email: email.toLowerCase(),
-            password: '12345Qwert!'
-          };
-        }
-      });
-    }
-  }
-});
+const faker = require('@faker-js/faker');
+const { defineConfig } = require("cypress");
+
+module.exports = defineConfig({
+  e2e: {
+    baseUrl: 'https://conduit.mate.academy/',
+    defaultCommandTimeout: 8000,
+    setupNodeEvents(on, config) {
+      on('task', {
+        generateUser() {
+          const email = faker.internet.email();
+          const randomNumber = Math.floor(Math.random(1000) * 1000);

```

Comment:
The random number generation logic `Math.floor(Math.random(1000) * 1000)` is incorrect. `Math.random()` generates a random number between 0 and 1, so the argument `1000` is unnecessary and does not affect the result. The correct way to generate a random number between 0 and 999 is `Math.floor(Math.random() * 1000)`.